algorithms/max-flow/problems/cowex/cowex.bats

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bats
+
+@test "Test in a graph without eddies" {
```

Comment:
If anything in the hack pack references them, then I think they should be explained somewhere.